vignettes/VCF.Rmd

```diff
@@ -15,7 +15,11 @@ knitr::opts_chunk$set(
 )
 ```
 
-## Call required libraries. Initialize gosling with `use_gosling()`
+This tutorial walks through the process of creating a Circos plot using VCF (Variant Call Format) data, with a focus on utilizing the shiny.gosling package to achieve interactive visualizations within a Shiny app.
```

Comment:
Following the same source you used in bigwig and other datatypes I suggest to add a reference for the VCF format as well
```suggestion
This tutorial walks through the process of creating a Circos plot using [VCF](https://genome.ucsc.edu/goldenPath/help/vcf.html) (Variant Call Format) data, with a focus on utilizing the shiny.gosling package to achieve interactive visualizations within a Shiny app.
```

Reply:
Done, updated.